tensorflow/c/c_api_experimental.h

```diff
@@ -314,7 +316,7 @@ TF_ImportGraphDefOptionsSetValidateColocationConstraints(
 // On success, place OK in status and return the newly created library handle.
```

Comment:
Nit: For method and function comments, please prefer using a verb with "this function" being an implicit subject. I.e., "On success, **returns** the newly created library handle and **places** OK in status"

Reply:
thanks for the comments, done.